gofmt.go

```diff
@@ -532,7 +528,49 @@ func gofmtMain(s *sequencer) {
 	}
 }
 
+type module struct {
+	Go     string
+	Module struct {
+		Path string
+	}
+}
+
+func loadModuleInfo(dir string) interface{} {
```

Comment:
any, or are you holding off on 1.18?

Reply:
Yes :) I support two Go versions by default on most of my projects, including this one